packages/destination-actions/src/destinations/facebook-custom-audiences/fbca-operations.ts

```diff
@@ -101,12 +104,14 @@ export const getApiVersion = (features?: Features, statsContext?: StatsContext):
 export default class FacebookClient {
   request: RequestClient
   adAccountId: string
+  features: Features | undefined
```

Comment:
nit: this is less verbose
```suggestion
  features?: Features
```